pages/index.js

```diff
@@ -620,7 +593,7 @@ export default class IndexPage extends React.Component {
       threshold: 0.1,
     };
 
-    const observer = new IntersectionObserver(function(entries, self) {
+    const observer = new IntersectionObserver(function (entries, self) {
```

Comment:
Extra whitespace here between anonymous `function` and `(entites`

Reply:
this is also the prettier auto-formatting, I can't do anything about it if I have prettier setup

Reply:
Hmm that is strange....